### DISCUSSION TITLE
Problemas con el componente Respuestas escritas tipo texto antiguos

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#126] [2. Questions and Answers] [answered]
Hola

Anteriormente para ese tipo de componente, existía el short-text  y el long-text. Actualmente sólo se encuentra uno vigente (short-text), dejándolo como texto visualmente. Ahora tengo el siguiente problema

Cuando quiero editar un survey que esté utilizando componente long-text y grabo, me arroja mensaje de error que no existe como subtipo. Entonces quisiera saber lo siguiente:

1.- ¿Es un bug?
2.- ¿Se debe pedir un feature?

Mi única solución sería editarlo por bd y revisar en todas las implementaciones los long-text existentes y cambiarlos por short-text para que no tenga problemas al editar desde el admin.

Adjunto imágenes y el código.

![image](https://user-images.githubusercontent.com/61419471/116602456-c330a400-a8f9-11eb-8398-894e30a15bb6.png)
![image](https://user-images.githubusercontent.com/61419471/116602469-c6c42b00-a8f9-11eb-954c-7d76d7a50a8b.png)

Código
```
{ 
    "_id" : ObjectId("5faa929ff617b300080b712b"), 
    "command" : {
        "commands" : null, 
        "isCommanded" : null, 
        "values" : [

        ]
    }, 
    "display" : [
        ""
    ], 
    "code" : [

    ], 
    "isActive" : true, 
    "isSystemModel" : false, 
    "contentType" : "application/vnd.cotalker.survey+textinput", 
    "help" : null, 
    "identifier" : "nombre_atencion_paciente", 
    "label" : "Nombre usuario", 
    "max" : NumberInt(5000), 
    "min" : NumberInt(0), 
    "required" : false, 
    "subtype" : "long_text", 
    "company" : ObjectId("5e0a1a1345e9602068206fc9"), 
    "modifiedAt" : ISODate("2020-12-21T19:25:22.709+0000"), 
    "__v" : NumberInt(0)
}```

**Comment by gmoplaza** [comment, marked as answer]
Es un bug, subo el fix en unos minutos.

**Reply by NicolasPLG** [reply]
gracias @gmoplaza 

Atte.
